### PR TITLE
enable ckeditor again for static content admin

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
fix https://github.com/symfony-cmf/sonata-phpcr-admin-integration-bundle/issues/89

Merge when https://github.com/symfony-cmf/sonata-phpcr-admin-integration-bundle/pull/100 is merged. This will enable the ck editor on static content admin again.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -136,11 +136,12 @@ cmf_sonata_phpcr_admin_integration:
         seo: ~
         menu: ~
         core: ~
-        content: ~
+        content:
```
Comment (dbu) [inline app/config/config.yml]: why do you remove the `~` here?

Comment (ElectricMaxxx) [inline app/config/config.yml]: had the configuration under `content:` on my first approach and moved to a global one then. So this change is wrong

File: composer.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
         "symfony-cmf/seo-bundle": "^2.0",
         "symfony-cmf/routing": "^2.0",
         "symfony-cmf/routing-auto-bundle": "^2.0",
-        "symfony-cmf/sonata-phpcr-admin-integration-bundle": "^1.0@dev",
+        "symfony-cmf/sonata-phpcr-admin-integration-bundle": "dev-fix_ckeditor",
```
Comment (dbu) [inline composer.json]: sandbox has a composer.lock file in version control so changing only this line does not do anything. no big deal, just saying.

File: app/config/config.yml (hunk 1 above)
Comment (ElectricMaxxx) [inline app/config/config.yml]: it is back again.

File: composer.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
         "doctrine/phpcr-odm": "^1.4",
 
         "sonata-project/cache-bundle": "^2.1",
-        "sonata-project/translation-bundle": "^2.0",
+        "sonata-project/translation-bundle": "2.1.0",
```
Comment (ElectricMaxxx) [inline composer.json]: see https://github.com/sonata-project/SonataTranslationBundle/issues/167

**Comment by dbu** [comment]
great! looking at travis-ci, seems we should use phpunit in require-dev and then run vendor/bin/phpunit

**Comment by ElectricMaxxx** [comment]
Looks green now. merged.

**Pull request merged**